src/cc/bpf_module_rw_engine.cc

```diff
@@ -416,7 +416,7 @@ int BPFModule::annotate() {
       StructType *st = dyn_cast<StructType>(t);
 #else
     if (PointerType *pt = dyn_cast<PointerType>(gvar->getType())) {
-      StructType *st = dyn_cast<StructType>(pt->getElementType());
+      StructType *st = dyn_cast<StructType>(pt->getPointerElementType());
```

Comment:
I think this will be addressed by #4020 .

Reply:
Thanks for guide! @chenhengqi :)

Reply:
Yes, just merged #4020, so the above change is not needed.